src/main/java/green/projectile/Projectile.java

```diff
@@ -33,5 +33,20 @@ public double getPeakY() {
         return ((velocity * Math.sin(angle)) * (velocity * Math.sin(angle))) / (2 * GRAVITY);
     }
 
+    public double getInterceptX() {
+        return getHorizontalVelocity() * getTime();
+    }
+
+    private double getHorizontalVelocity() {
+        return velocity * Math.cos(angle);
+    }
 
+    private double getVerticalVelocity() {
+        return velocity * Math.sin(angle);
+    }
+
+    private double getTime() {
```

Comment:
Put in a comment explaining what this method is because getTime() is very unspecific for a class that already has a seconds field.